ccv/chains/evm/deployment/v1_7_0/changesets/deploy_chain_contracts.go

```diff
@@ -10,8 +11,9 @@ import (
 )
 
 type DeployChainContractsCfg struct {
-	ChainSel uint64
-	Params   sequences.ContractParams
+	ChainSel        uint64
+	ContractFactory common.Address
```

Comment:
Contract factory will be passed in as a raw address as it will be living in a different environment in CLD